manifests/m/Microsoft/PowerShell/Preview/7.3.7.0/Microsoft.PowerShell.Preview.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell.Preview
+PackageVersion: 7.3.7.0
```

```suggestion
PackageVersion: 7.3.0.7
```
Looks like a typo? because this must match Control Panel to prevent upgrade loops.